elmslie-android/src/main/java/vivid/money/elmslie/android/base/ElmActivity.kt

```diff
@@ -22,6 +23,14 @@ abstract class ElmActivity<Event : Any, Effect : Any, State : Any> :
         get() = storeHolder.store
 
     override val storeHolder: StoreHolder<Event, Effect, State> by fastLazy {
-        LifecycleAwareStoreHolder(lifecycle) { createStore()!! }
+        createStoreHolder { createStore() }
     }
+
+    override fun createStoreHolder(
```

Comment:
kinda complicated:
we have the property `storeHolder` and overrided method `createStoreHolder`, I guess we would leave one of them

Reply:
We need it to have a default implementation for createStoreHolder that is forced to use ::createStore. With current approach, you can override storeHolder and forget to use createStore method.